assert_release_quality

```diff
@@ -1,8 +1,20 @@
 #!/bin/sh
+#
+# Asserts that the module is ready for release with the expected quality
 
 # cf. https://docs.npmjs.com/cli/version
 new_version=$1
 
+# Checking that the npm-shrinkwrap.json file is up-to-date wrt to the
+# package.json file.
+if [ package.json -nt npm-shrinkwrap.json ]
+then
+    echo "File package.json is newer than file npm-shrinkwrap.json"
```

Comment:
A solution is to move the previous `npm-shrinkwrap.json` file, generate a new one and then compare the two to see if something has changed, but that would be more complicated to understand at first sight.